projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -433,14 +433,14 @@ private function parse_menu_item( $title ) {
 		if ( false !== strpos( $title, 'count-' ) ) {
 			preg_match( '/<span class=".+\s?count-(\d*).+\s?<\/span><\/span>/', $title, $matches );
```

Comment:
For all of these, consider doing like this instead
```suggestion
		if ( false !== strpos( $title, 'count-' ) &&
			preg_match( '/<span class=".+\s?count-(\d*).+\s?<\/span><\/span>/', $title, $matches )
		) {
```
i.e. don't even enter the `if` if the regex match fails.

Possibly we could even simplify it futher, like
```suggestion
		if ( preg_match( '/<span class=".+\s?count-(\d*).+\s?<\/span><\/span>/', $title, $matches ) ) {
```
unless there was a performance reason it wasn't done that way in the first place in #17594 rather than just not knowing that `preg_match()` returns a boolean indicating whether there were any matches.

Reply:
It looks like the code is being ran in a loop for every menu and submenu item.
I would keep the `strpos` in place for performance.